content/minfraud/api-documentation/_schemas/RequestAccount.mdx

```diff
@@ -20,9 +20,13 @@ import requestJson from '../_examples/request';
     login name for the account, but rather should be an internal ID that does
     not change. This is not your MaxMind account ID. No specific format is
     required.
+
+    [Learn more about the `/account/user_id` input on our Knowledge
+    Base.](https://support.maxmind.com/hc/en-us/articles/5452006111003#h_01G1E0XKAEF0BBC336545SY2AA)
   </Property>
 
   <Property
+    isDeprecated
```

Comment:
Is this really deprecated?

Reply:
yes, thomas recently asked us to deprecate this input